blockly-core/core/ui/block_space/horizontal_flyout.js

```diff
@@ -32,6 +32,18 @@ Blockly.HorizontalFlyout.prototype.show = function(xmlList) {
   Blockly.HorizontalFlyout.superClass_.show.apply(this, arguments);
 };
 
+Blockly.HorizontalFlyout.prototype.setVisibility = function(show) {
+  return show ? this.softShow() : this.softHide()
+};
```

Comment:
does return value have any meaning here? might also be worth a comment about what "soft" means